lib/command.js

```diff
@@ -106,6 +106,10 @@ class Command extends EventEmitter {
     this._showHelpAfterError = sourceCommand._showHelpAfterError;
     this._showSuggestionAfterError = sourceCommand._showSuggestionAfterError;
 
+    this.commands.forEach(command => {
+      command.copyInheritedSettings(this);
```

Comment:
Somewhat cosmetic after the copy, but I suggest: 
```suggestion
      command.copyInheritedSettings(sourceCommand);
```